src/Reader.php

```diff
@@ -91,6 +92,7 @@ public function read($import, $filePath, string $readerType, string $disk = null
             $sheet->disconnect();
         }
 
+        $this->afterReading($import);
```

Comment:
For consistency, can you call this `afterImport()`

Reply:
The reason I chose "afterReading" is that there is already a method on this class called "beforeReading"  that triggers the "beforeImport" event (as well as other functions) and I wanted to make it consistent with that. 

While this method as I wrote it simply raises an event, I could see future contributors adding other things to happen after the reading so in the name of consistency I want to keep it open.

Reply:
If it does hold back putting in the pull request I don't have any objections to changing it to afterImport, but I did want to explain my rationale for the naming convention I used.

Reply:
I get it. Can you move the garbage collecting call into the after reading. Make sure it's called after raising the event. Thanks!